### ISSUE TITLE
An error occured while receiving wifi notification for input string "f0157b65854f00e5"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CarlitoBriganteNT/android-notifier#232]
```
What steps will reproduce the problem?
1. Send test notification from android phone

What is the expected output? What do you see instead?
I get a pop-up error message immediately on my Ubuntu desktop:
An error occured while receiving wifi notification: For input string 
"f0157b65854f00e5"

What version of the product are you using? On what operating system?
Ubuntu Desktop Notifier: android-notifier-desktop_0.5-0_amd64.deb
Android app:  0.2.8

Installed ubuntu application by adding google code repository.
```

Original issue reported on code.google.com by `kj4...@gmail.com` on 13 Oct 2010 at 12:27
- Merged into: #209

**Comment by GoogleCodeExporter** [comment]
```
Could you please attach your log file in this issue?
```

Original comment by `lehph...@gmail.com` on 13 Oct 2010 at 2:05
- Added labels: **OpSys-Linux**

**Comment by GoogleCodeExporter** [comment]
```
Oops, ment to do that in the original post ;)
```

Original comment by `kj4...@gmail.com` on 13 Oct 2010 at 2:09

Attachments:
- [android-notifier-desktop.log](https://storage.googleapis.com/google-code-attachments/android-notifier/issue-232/comment-2/android-notifier-desktop.log)

**Comment by GoogleCodeExporter** [comment]
```
Ok, this issue has been fixed in release 0.5.1, please upgrade.
```

Original comment by `lehph...@gmail.com` on 13 Oct 2010 at 2:14
- Changed state: **Duplicate**
